bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -153,16 +154,25 @@ if args.max_bin_param:
     pbin_upper_lim = float(args.max_bin_param)
 else:
     pbin_upper_lim = params[args.bin_param].max()
-bb_input = (params[args.bin_param].min(), pbin_upper_lim, args.num_bins)
+
+
+if args.bin_param == 'template_duration' and params[args.bin_param].min() == 0:
```

Comment:
should it be clear why having min duration of 0 indicates that some templates are empty?  (at least I don't immediately see that)

Reply:
Comment added, hopefully it is clear

Reply:
not really, i don't see what is meant .. 'dereferencing' is something you do with pointers in C afaik, and where above? 
if you mean a previous line of code, probably better to explain it there (ie say what gives rise to the zero value)